src/windowmanagement/TaskBarMenuChoices.tsx

```diff
@@ -27,37 +32,51 @@ export function TaskBarMenuChoices({
       arrowPosition={'center'}
     >
       <Menu.Target>
-        <Menu.Item>Task Bar</Menu.Item>
+        <Menu.Item
+          leftSection={<TaskBarIcon />}
+          rightSection={<ChevronRightIcon size={IconSize.sm} />}
+        >
+          Task Bar
+        </Menu.Item>
       </Menu.Target>
       <Menu.Dropdown>
         <Menu.Label>Toggle Task Bar Items</Menu.Label>
-        <Stack gap={'xs'}>
-          {menuItemsDB.map((item) => (
-            <Checkbox
-              key={item.componentID}
-              label={item.title}
-              defaultChecked={checkedMenuItems[item.componentID]}
-              onChange={(event) => {
-                const { checked } = event.currentTarget;
-                setVisibleMenuItems((prevstate) => {
-                  if (checked) {
-                    // This check is necessary because the callback is sometimes called
-                    // twice, meaning we would otherwise add the item multiple times
-                    if (prevstate.includes(item.componentID)) {
-                      return prevstate;
-                    }
-                    prevstate.push(item.componentID);
-                  } else {
-                    const index = prevstate.findIndex((id) => id === item.componentID);
-                    if (index >= 0) {
-                      prevstate.splice(index, 1);
-                    }
-                  }
-                  return [...prevstate];
-                });
-              }}
-            />
-          ))}
+        <Stack gap={0}>
```

Comment:
Why are we not using `Menu.Item` here?  Using a stack of groups, we lose the navigation behavior that there are in other menus (arrow up/down)

Reply:
I'm not sure, but it might be because the `Menu.Item` acts as a button and so that would mess with the `Checkbox`. But I'll have to go back and check again, this was done a loooong time ago